apps/src/music/utils.js

```diff
@@ -0,0 +1,7 @@
+// This is mostly to separate the 'blockly/media' path from music lab code.
```

Comment:
@madelynkasula @molly-moen just FYI, created this quick util to remove 'blockly/media' specific paths out of music lab. I figured replacing our existing references to 'blockly/media' would be something we'd want to look at in lab lab?